### ISSUE TITLE
WCAG: Miles-banner, image should be coded as image

### ISSUE THREAD

**Issue by knut-hufthammer** [miles-no/miles_web_components#114]
The image in the banner is a background on a div. It should be coded as an image and preferably have alt-text. An alternative could be to add role="presentation".

![Image](https://user-images.githubusercontent.com/1680898/258766089-877f8fce-e769-4ae6-941f-0f50b87b15c9.png)

**Comment by knut-hufthammer** [comment]
This was suggested by Paal Kristian back in August. We should prioritize this now to better comply with WCAG requirements.
